www/docs/errors.md

```diff
@@ -33,28 +33,62 @@ These errors are displayed on the terminal.
 
 #### OAUTH_GET_ACCESS_TOKEN_ERROR
 
+This occurs when there was an error in the POST request to the OAuth provider and we were not able to retrieve the access token.
+
+Please double check your provider settings.
+
 #### OAUTH_V1_GET_ACCESS_TOKEN_ERROR
 
+This error is explicitly related to older OAuth v1.x providers, if you are using one of these, please double check all available settings.
+
 #### OAUTH_GET_PROFILE_ERROR
 
+N/A
+
 #### OAUTH_PARSE_PROFILE_ERROR
 
+This error is a result of either a problem with the provider response or the user cancelling the action with the provider, unfortunately we can't discern which with the information we have.
+
+This error should also log the exception and available `profileData` to further aid debugging.
+
 #### OAUTH_CALLBACK_HANDLER_ERROR
 
+This error will occur when there was an issue parsing the json request body, for example.
+
+There should also be further details logged when this occurs, such as the error thrown, and the request body itself to aid in debugging.
+
 ---
 
 ### Signin / Callback
 
 #### GET_AUTHORIZATION_URL_ERROR
 
+This error can occur when we cannot get the OAuth v1 request token and generate the authorization URL.
+
+Please double check your OAuth v1 provider settings, especially the OAuth token and OAuth token secret.
+
 #### SIGNIN_OAUTH_ERROR
 
+This error can occur in one of a few places, first during the redirect to the authorization URL of the provider. Next, in the signin flow while creating the PKCE code verifier. Finally, during the generation of the CSRF Token hash in internal state during signin.
+
+Please check your OAuth provider settings and make sure your URLs and other options are correctly set on the provider side.
+
 #### CALLBACK_OAUTH_ERROR
 
+This can occur during handling of the callback if the `code_verifier` cookie was not found or an invalid state was returned from the OAuth provider.
+
 #### SIGNIN_EMAIL_ERROR
 
+This error can occur if there was an error signing in a user via Email link, such as if the email token could not be generated or the verification request failed.
```

Comment:
```suggestion
This error can occur when a user tries to sign in via an email link; for example, if the email token could not be generated or the verification request failed.
```
_I think_ email should be in lowercase here and re-wrote the sentence for clarity 👍🏽

Reply:
Yeah nice improvement!